tests/test_genesis.py

```
@@ -0,0 +1,85 @@
+
+
+def test_genesis_requires_empty_state() -> None:
+    config = GenesisConfig(initial_balances={"dd" * 20: 1})
+    block = create_genesis_block(config)
+    state = State()
+    state.set_account("ff" * 20, Account(balance=1, nonce=0))
+
+    try:
+        apply_genesis_block(state, block, config)
+    except ValueError as exc:
+        assert "empty state" in str(exc)
```

Ruff (0.15.1)

[warning] Found assertion on exception `exc` in `except` block, use `pytest.raises()` instead

(PT017)